grapple/types/streamfield.py

```diff
@@ -329,17 +331,50 @@ def get_media_url(url):
     return url
 
 
+def get_embed_url(instance):
+    return instance.value.url if hasattr(instance, "value") else instance.url
+
+
+def get_embed_object(instance):
+    try:
+        return get_embed(get_embed_url(instance))
+    except EmbedException:
+        pass
+
+
 class EmbedBlock(graphene.ObjectType):
     value = graphene.String(required=True)
     url = graphene.String(required=True)
+    embed = graphene.String()
+    raw_embed = graphene.JSONString()
 
     class Meta:
         interfaces = (StreamFieldInterface,)
 
     def resolve_url(self, info, **kwargs):
-        if hasattr(self, "value"):
-            return get_media_url(self.value.url)
-        return get_media_url(self.url)
+        return get_media_url(get_embed_url(self))
+
+    def resolve_raw_value(self, info, **kwargs):
```

Comment:
Did you forget to add the `raw_value` GraphQL field on this type? 😉

Reply:
No, that is inherited from `StreamFieldInterface`, however I did discover I imported the wrong `EmbedValue` so this was erroring again 🤦 (before this, it would fail with `EmbedValue has no attribute 'value'` via `StreamFieldInterface.resolve_raw_value`)

Reply:
Oh, makes sense. Looks good to go.